<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Modal</class>
 <widget class="QDialog" name="Modal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>617</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pointsButton">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>130</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter_4">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>230</y>
     <width>186</width>
     <height>28</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QPushButton" name="saveButton">
    <property name="text">
     <string>Сохранить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="discardButton">
    <property name="text">
     <string>Отмена</string>
    </property>
   </widget>
  </widget>
  <widget class="QSplitter" name="splitter_3">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>166</y>
     <width>581</width>
     <height>51</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="text">
     <string>Поле main_table.t</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="m_t_lineEdit">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>581</width>
      <height>28</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>581</width>
      <height>28</height>
     </size>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>41</y>
     <width>581</width>
     <height>51</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="text">
     <string>Поле main_table.i</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="m_i_lineEdit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>581</width>
      <height>28</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>581</width>
      <height>28</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>581</width>
      <height>28</height>
     </size>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
   </widget>
  </widget>
  <widget class="QSplitter" name="splitter_2">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>104</y>
     <width>481</width>
     <height>51</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="text">
     <string>Поле qual_table.p01 связанное через main_table.r с qual_table.p00</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="m_r_lineEdit">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>481</width>
      <height>28</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>481</width>
      <height>28</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
